app/payloads/api/v1/patient_payload_validator.rb

```diff
@@ -16,14 +16,21 @@ class Api::V1::PatientPayloadValidator < Api::V1::PayloadValidator
 
   validate :validate_schema
   validate :presence_of_age
+  validate :past_date_of_birth
 
   def presence_of_age
     unless date_of_birth.present? || (age.present? && age_updated_at.present?)
       errors.add(:age, 'Either date_of_birth or age and age_updated_at should be present')
     end
   end
 
+  def past_date_of_birth
```

Comment:
This is fine, but it seems like we should delegate these kinds of validations to the model somehow so we don't repeat code here.